tests/test_api.py

```diff
@@ -290,15 +371,60 @@ def test_get_build_api(self, osbs):
         # We should get a BuildResponse
         assert isinstance(response, BuildResponse)
 
-    def test_get_build_request_api(self, osbs):
-        build = osbs.get_build_request()
+    @pytest.mark.parametrize('build_type', (
+        None,
+        'simple',
+        'prod',
+        'prod-without-koji'
+    ))
+    def test_get_build_request_api_build_type(self, osbs, build_type):
```

Comment:
can we add comment that we are just testing if providing deprecated 'build_type' won't break anything?